packages/pynumaflow/examples/sink/all_sinks/example.py

```diff
@@ -0,0 +1,79 @@
+import os
+from collections.abc import AsyncIterable
+from pynumaflow.sinker import Datum, Responses, Response, Sinker, Message
+from pynumaflow.sinker import SinkAsyncServer
+import logging
+import random
+
+logging.basicConfig(level=logging.DEBUG)
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Just use `logger` for the variable name. This is an application (binary), no need to mark the variable as private. Also, capital letters are usually for constants.